mkosi.default.d/fedora/10-mkosi.fedora

```diff
@@ -75,3 +75,10 @@ Packages=
         netcat
         e2fsprogs
         compsize
+        # Required to run systemd-networkd-tests.py
+        iproute
```

Comment:
Preferably in a separate PR :p. I don't mind adding the packages, but testing everything actually works takes quite a bit of time so I'd rather leave that to someone that actually uses the distro.

Reply:
yeah I wasn't asking to make sure it works, simply to keep the installed packages in sync (to avoid surprises) - for debian and ubuntu, this should be the list:

iproute2
dnsmasq-base
wireguard-tools
isc-dhcp-server

Reply:
Oops, I forgot this comment and merged this PR. Let's do that later.